src/entities/mellowLpRouter.ts

```diff
@@ -445,45 +393,51 @@ class MellowLpRouter {
     if (
       isUndefined(this.readOnlyContracts) ||
       isUndefined(this.writeContracts) ||
-      isUndefined(this.userAddress) ||
-      isUndefined(this.erc20RootVaultInfo)
+      isUndefined(this.userAddress)
     ) {
       throw new Error('Uninitialized contracts.');
     }
 
-    if (this.readOnlyContracts.erc20RootVault.length > 1) {
```

Comment:
Here I've changed the logic a bit. Let's restrict to pivoted routers for now. The pivot is going to be the vault index where we withdraw from.